app/Http/Controllers/Auth/AuthController.php

```diff
@@ -38,40 +36,38 @@ public function getCheckpoint()
 
     public function getIndex()
     {
+        $state = str_random(5);
```

Comment:
That's fine. it may generate same string but the sessions are different for each users